### ISSUE TITLE
seq2seq.sequence_loss

### ISSUE THREAD

**Issue by shiningliang** [ProHiryu/bert-chinese-ner#32]
看到代码中有这一段

mask = tf.cast(input_mask,tf.float32)

loss = tf.contrib.seq2seq.sequence_loss(logits,labels,mask)

return (loss, logits, predict)

请问为何被注释后换成了自己手写的损失函数，使用这个带mask的sequence loss有什么问题吗？

**Comment by ProHiryu** [comment]
没什么问题
主要原因在于想得到per_example_loss用作eval，更加清晰和方便

**Comment by shiningliang** [comment]
非常感谢！
